src/main/java/duke/command/ListCommand.java

```diff
@@ -0,0 +1,37 @@
+package duke.command;
+
+import duke.MultiLineFormatter;
+import duke.Storage;
+import duke.task.Task;
+import duke.TaskList;
+import duke.Ui;
+
+/**
+ * ListCommand class represents the list command given by the user.
+ */
+public class ListCommand extends Command {
+
+    /**
+     * Returns the string representation of
+     * a list of tasks that are in the task list.
+     *
+     * @param ui Ui object which handles the interaction with the user.
+     * @param storage Storage object which handles interaction with data in file.
+     * @param taskList List of tasks.
+     * @return String representing list of tasks.
+     */
+    @Override
+    public String execute(Ui ui, Storage storage, TaskList taskList) {
+        MultiLineFormatter mf = new MultiLineFormatter();
```

Comment:
I understand that the word "multilineformatter" can be too long to use as a variable name. Perhaps you could have the best of the both worlds such as naming it as "MFormatter"?